casts/cards/src/UICollectionCard/types.ts

```diff
@@ -0,0 +1,25 @@
+import type React from 'react';
+import type { UIImageProps } from '@tonlabs/uikit.media';
+import type { ViewStyle, ImageSourcePropType } from 'react-native';
+
+export type UICollectionCardProps = {
+    title?: string;
+    badge?: number | React.ReactElement<UIImageProps>;
+    onPress?: () => void;
+    imageSourceList?: ImageSourcePropType[];
+    loading?: boolean;
+    testID?: string;
+};
+
+export type QuickViewProps = {
+    imageSourceList?: ImageSourcePropType[];
+    style: ViewStyle;
+};
+
+export type TitleProps = {
+    title?: string;
+};
+
+export type BadgeProps = {
+    badge?: number | React.ReactElement<UIImageProps>;
```

Comment:
what do you think instead of `React.ReactElement<UIImageProps>` pass `UIImageProps['source']`, and set all needed styles (especially width and height) themselves? This way all a developer need to do is to just import from assets and pass it